qemu/tests/cfg/nic_acpi_index.cfg

```diff
@@ -0,0 +1,26 @@
+- nic_acpi_index:
+    virt_test_type = qemu
+    type = nic_acpi_index
+    only Linux
+    no q35
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    nic_hotplug_count=1
+    pci_model = virtio-net-pci
```

Comment:
`nic_model = virtio-net-pci`?

Reply:
@PaulYuuu I have changed it. And sent another patch to support this change.